.azure-pipelines-templates/daily-matrix.yml

```diff
@@ -19,24 +19,37 @@ parameters:
       cmake_args: '-DCMAKE_INSTALL_PREFIX=/tmp/ccf-install'
 
 jobs:
-  - template: common.yml
-    parameters:
-      target: NoSGX
-      env: '${{ parameters.env.NoSGX }}'
-      cmake_args: '${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
-      suffix: 'Coverage'
-      artifact_name: 'NoSGX_Coverage'
-      ctest_filter: '-LE "benchmark|perf|long_test"'
-      ctest_timeout: '300'
+  - job: 'Make Generator'
+    env: '${{ parameters.env.NoSGX }}'
+    variables:
+      skipComponentGovernanceDetection: true
+    steps:
+      - checkout: self
+        clean: true
+      - script: |
+          mkdir build_make
+          cd build_make
+          cmake -L ..
+        displayName: CMake
 
-  - template: common.yml
-    parameters:
-      target: SGX
-      env: '${{ parameters.env.SGX }}'
-      cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} -DTLS_TEST=ON -DSHUFFLE_SUITE=ON'
-      suffix: 'Release'
-      artifact_name: 'SGX_Release'
-      ctest_filter: '-LE "benchmark|perf"'
-      ctest_timeout: '240'
-      install_prefix: '${{ parameters.build.install.install_prefix }}'
-      tls_report: true
\ No newline at end of file
+  # - template: common.yml
+  #   parameters:
+  #     target: NoSGX
+  #     env: '${{ parameters.env.NoSGX }}'
+  #     cmake_args: '${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
+  #     suffix: 'Coverage'
+  #     artifact_name: 'NoSGX_Coverage'
+  #     ctest_filter: '-LE "benchmark|perf|long_test"'
+  #     ctest_timeout: '300'
+
+  # - template: common.yml
+  #   parameters:
+  #     target: SGX
+  #     env: '${{ parameters.env.SGX }}'
+  #     cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} -DTLS_TEST=ON -DSHUFFLE_SUITE=ON'
+  #     suffix: 'Release'
+  #     artifact_name: 'SGX_Release'
+  #     ctest_filter: '-LE "benchmark|perf"'
+  #     ctest_timeout: '240'
+  #     install_prefix: '${{ parameters.build.install.install_prefix }}'
+  #     tls_report: true
```

Comment:
```suggestion
  - template: common.yml
    parameters:
      target: NoSGX
      env: '${{ parameters.env.NoSGX }}'
      cmake_args: '${{ parameters.build.debug.cmake_args }} ${{ parameters.build.NoSGX.cmake_args }}'
      suffix: 'Coverage'
      artifact_name: 'NoSGX_Coverage'
      ctest_filter: '-LE "benchmark|perf|long_test"'
      ctest_timeout: '300'

  - template: common.yml
    parameters:
      target: SGX
      env: '${{ parameters.env.SGX }}'
      cmake_args: '${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} -DTLS_TEST=ON -DSHUFFLE_SUITE=ON'
      suffix: 'Release'
      artifact_name: 'SGX_Release'
      ctest_filter: '-LE "benchmark|perf"'
      ctest_timeout: '240'
      install_prefix: '${{ parameters.build.install.install_prefix }}'
      tls_report: true
```

Reply:
Done - just had these disabled temporarily to test the new job in isolation.